pyston/resource.py

```diff
@@ -179,13 +180,22 @@ class BaseResource(six.with_metaclass(ResourceMetaClass, PermissionsResourceMixi
         'content_type': ('CONTENT_TYPE', '_content_type'),
     }
 
+    DATA_KEY_MAPPING = {}
+
     def __init__(self, request):
         self.request = request
         self.args = []
         self.kwargs = {}
 
+    def _demap_key(self, lookup_key):
+        real_key = lookup_key
+        for key, value in self.DATA_KEY_MAPPING.items():
```

Comment:
co takto?

return {v: k for k, v in self.DATA_KEY_MAPPING.items()}.get(lookup_key)